src/convertToObject.js

```diff
@@ -3,16 +3,31 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles(seeanexamplein[stylesString.js]
+ * (./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the valuesofrelated CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleArray = sourceString.split(';').filter(Boolean);
+
+  const stylesObject = {};
+
+  styleArray.forEach(styleDeclaration => {
+    const [property, value] = styleDeclaration.split(':').map(part =>
```

Comment:
helpful advice, please fix
![image](https://github.com/mate-academy/js_style-to-object/assets/54936044/dd39ba49-ee21-43a8-9921-44f3cecc0dfb)